docs/reference/search/search-your-data/semantic-search.asciidoc

```diff
@@ -7,108 +7,89 @@ Semantic search is a search method that helps you find data based on the intent
+| <<semantic-text,`semantic_text`>>      | <<query-dsl-semantic-query,`semantic`>>           | The `semantic_text` field handles generating embeddings for you at index time and query time.                                                               
+| <<sparse-vector,`sparse_vector`>>      | <<query-dsl-sparse-vector-query,`sparse_vector`>> | The `sparse_vector` query can generate query embeddings for you, but you can also provide your own. Provide embeddings at index time.
+| <<dense-vector,`dense_vector`>>        | <<query-dsl-knn-query,`knn`>>                     | The `knn` query can generate query embeddings for you, but you can also provide your own. Provide embeddings at index time.
+|=======================================================================================================================================================================================================
 
-[discrete]
-[[semantic-search-hybrid-search]]
-=== Beyond semantic search with hybrid search
+If you want {es} to generate embeddings at both index and query time, use the `semantic_text` field and the `semantic` query.
+If you want to bring your own embeddings, store them in {es} and use the `sparse_vector` or `dense_vector` field type and the associated query depending on the NLP model you used for generating the embeddings.
 
-In some situations, lexical search may perform better than semantic search.
-For example, when searching for single words or IDs, like product numbers.
-
-Combining semantic and lexical search into one hybrid search request using
-<<rrf,reciprocal rank fusion>> provides the best of both worlds.
-Not only that, but hybrid search using reciprocal rank fusion {blog-ref}improving-information-retrieval-elastic-stack-hybrid[has been shown to perform better in general].
+IMPORTANT: For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.
 
-include::{es-ref-dir}/tab-widgets/semantic-search/hybrid-search-widget.asciidoc[]
 
 [discrete]
 [[semantic-search-read-more]]
 === Read more
 
 * Tutorials:
-** <<semantic-search-elser,Semantic search with ELSER>>
+** <<semantic-search-semantic-text, Semantic search with `semantic_text`>>
+** <<semantic-search-inference, Semantic search with the {infer} API>>
+** <<semantic-search-elser,Semantic search with ELSER>> using the {infer} workflow
+** <<semantic-search-deployed-nlp-model, Semantic search with a model deployed in {es}>>
 ** {ml-docs}/ml-nlp-text-emb-vector-search-example.html[Semantic search with the msmarco-MiniLM-L-12-v3 sentence-transformer model]
+* Interactive examples:
+** The https://github.com/elastic/elasticsearch-labs[`elasticsearch-labs`] repo contains a number of interactive semantic search examples in the form of executable Python notebooks, using the {es} Python client
 * Blogs:
```

Comment:
Can we link to the [`semantic_text` blog post](https://www.elastic.co/search-labs/blog/semantic-search-simplified-semantic-text) in this section?